go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/hashicorp/nomad-driver-virt
 
-go 1.23
+go 1.23.0
```

Comment:
Is this a change we need? If so, do we need it to match the Go version specified within the `.go-version` file?

Reply:
This is just to get the toolchain working on my local, fixing it